packages/react-theme/config/api-extractor.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "@fluentui/scripts/api-extractor/api-extractor.common.json"
```

Comment:
I've added this file manually based on current plop template.

Reply:
https://github.com/microsoft/fluentui/issues/18658